public/main/plugins/ethWallet/index.js

```diff
@@ -363,6 +363,7 @@ function syncTransactions ({ number, walletId, webContents, bloqEthExplorer }) {
       logger.debug('Syncing', addresses, indexed)
       return Promise.all(
         addresses.map(function (address) {
+          webContents.send('transactions-scan-started', { address })
```

Comment:
@gabmontes check the changes in this file!
Seemed simple enough to add the required messages, but maybe I overlooked something...

Reply:
Shouldn't we consider the case the `'transactions-scan-started'` is sent, then the promise chain breaks and the `'transactions-scan-finished'` is never sent? Perhaps adding that event to the final `catch` down below.

Reply:
good `catch`! (pun intended)